app/models/schools/configuration.rb

```diff
@@ -0,0 +1,80 @@
+# == Schema Information
+#
+# Table name: configurations
+#
+#  created_at       :datetime         not null
+#  dashboard_charts :json             not null
+#  id               :bigint(8)        not null, primary key
+#  school_id        :bigint(8)        not null
+#  updated_at       :datetime         not null
+#
+# Indexes
+#
+#  index_configurations_on_school_id  (school_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (school_id => schools.id) ON DELETE => cascade
+#
+
+module Schools
+  class Configuration < ApplicationRecord
+    belongs_to :school
+
+    delegate :meters, :meters_with_validated_readings, to: :school
+
+    def fuel_types_for_analysis
+      if is_school_dual_fuel?
+        dual_fuel_fuel_type
+      elsif meters_with_validated_readings(:electricity).exists?
+        electricity_fuel_type
+      elsif meters_with_validated_readings(:gas).exists?
+        :gas_only
+      else
+        :none
+      end
+    end
+
+    # Duplicating school method temporarily
+    def has_solar_pv?
+      meters.detect(&:solar_pv?)
+    end
+
+    # Duplicating school method temporarily
+    def has_storage_heaters?
+      meters.detect(&:storage_heaters?)
+    end
+
+    def dashboard_charts_symbols
```

Comment:
The use of dashboard is a bit confusing here, as these aren't charts for the dashboard.

Reply:
Analysis charts - rather than dashboard charts as they are what we now call the pupil and teacher dashboards...